src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -21,11 +21,12 @@ export class ErrorBoundary extends Component<unknown, State> {
   }
 
   render(): ReactNode {
-    const error = `(${this.state.errorMessage} ${formatDateTime(Date.now())})`;
+    const error = `(${this.state.errorName} ${formatDateTime(Date.now())})`;
```

Comment:
Noticed that we're now sending the `errorName` down to `ErrorBoundaryContent`, shall we let it handle the formatting of the `error` string as well?